.github/workflows/aks.yaml

```diff
@@ -94,9 +94,6 @@ jobs:
           cilium install \
             --cluster-name=${{ env.name }} \
             --azure-resource-group ${{ env.name }} \
-            --azure-tenant-id ${{ secrets.AZURE_PR_SP_TENANT_ID }} \
-            --azure-client-id ${{ secrets.AZURE_PR_SP_CLIENT_ID }} \
-            --azure-client-secret ${{ secrets.AZURE_PR_SP_CLIENT_SECRET }} \
```

Comment:
What's the implication of this change exactly? Does that mean a new service principal is created for each new test or was that already the case?

Reply:
A new dedicated SP is created for each test:
https://github.com/cilium/cilium-cli/blob/b70e437690f97110e43df96e4cffedff273ab131/install/azure.go#L81-L100

This is also how most users will/should be using `cilium install`, so this also makes testing closer to what users will actually use.